src/js/angular/guides/guides.service.js

```diff
@@ -162,6 +189,12 @@ function GuidesService($http, $rootScope, $translate, $interpolate, ShepherdServ
         this._subscribeToGuideResumed();
         this._subscribeToGuideCancel();
         this._subscribeToGuidePause();
+        $rootScope.downloadGuidesFile = (resourcePath, resourceFile) => {
```

Comment:
No! This won't make it! Better move this function in the main controller.

Reply:
Moved